frontend/src/components/pages/CardDetails/Presentation.jsx

```diff
@@ -19,33 +19,66 @@ import AssigneesList from "../../widgets/AssigneesList";
 import ReviewersTable from "../../widgets/ReviewersTable";
 import Reviews from "./Reviews";
 
-const useStyles = makeStyles((theme) => {
-  return {
-    paper: {
-      padding: theme.spacing(1, 2, 1),
-    },
+const TableCell = withStyles({
+  root: {
+    borderBottom: "none",
+    padding: "3.24%",
+  },
+})(MuiTableCell);
 
-    commentColumn: {
-      minWidth: 300,
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
     },
-
-    tableContainer: {
-      maxHeight: 200,
+    paper: {
```

Comment:
Is the `paper` class no longer being used? Have you considered removing unused things?